futures-sink/Cargo.toml

```diff
@@ -11,9 +11,9 @@ The asynchronous `Sink` trait for the futures-rs library.
 """
 
 [features]
-std = ["futures-core/std"]
+std = ["futures-core/std", "futures-channel/std"]
```

Comment:
Ditto question about `futures-channel/std`-- this crate currently doesn't have any non-std behavior. What should we do here?

Reply:
Hm this actually makes me think that we may want to have a `std` feature even for the `futures-channel` crate (which makes the crate compile to nothing). That way crates like `futures-sink` which are actually somewhat `no_std` compatible would have a way of saying "you just don't get channel things if you turn off `std`".

How's that sound?

Reply:
Sounds good to me!